sympy/physics/control/lti.py

```diff
@@ -1656,31 +1656,61 @@ def __neg__(self):
         return MIMOParallel(*arg_list)
 
 
-class Feedback(Basic):
-    """
-    A class for representing negative feedback interconnection between two
-    input/output systems. The first argument, ``num``, is called as the
-    primary plant or the numerator, and the second argument, ``den``, is
-    called as the feedback plant (which is often a feedback controller) or
-    the denominator. Both ``num`` and ``den`` can either be ``Series`` or
+class Feedback(SISOLinearTimeInvariant):
+    r"""
+    A class for representing closed-loop feedback interconnection between two
+    SISO input/output systems. The first argument, ``plant``, is the
+    primary plant of the closed-loop system or in simple words, the transfer
+    function representing the system/process to be controlled. Most of the time, a controller system is also
+    placed in series with the plant. This controller is known as the **plant controller** as it controls
+    the plant by varying the input that goes into plant. The second argument, ``controller``,
+    as the name suggests, controls the fed back signal to the ``plant``. It is the equivalent transfer
+    function of the feedback controling system which is generally a sensor system that constantly
+    takes in the output signal produced and feeds it back to the primary plant (or a plant-controller
+    if one is connected in series with the plant). Note that the feedback controller is not same as the
+    plant controller.
+
+    Generally, in control theory, we want the output signal to follow the input signal or in
```

Comment:
Just mention that you can change the feedback sign leading to either direct sum or a negated sum. This paragraph is a bit wordy.